repositories/cyclonedds.BUILD.bazel

```diff
@@ -179,6 +179,10 @@ cmake(
         },
         no_match_error = "Unsupported build configuration",
     ),
+    copts = [
+        "-Wno-documentation-deprecated-sync",
+        "-Wno-error",
```

Comment:
Which compiler complains about this? Shall we just set `-w`?

Reply:
LLVM 19, I am fine with `-w` as well.

Reply:
OK, let's go with "-w" then.

Reply:
Done.